ui/admin/app/templates/scopes/scope/workers/worker/tags.hbs

```diff
@@ -64,25 +64,101 @@
             </B.Td>
             <B.Td>
               {{#if (eq B.data.type this.tagTypeApi)}}
-                <Hds::Dropdown as |dd|>
-                  <dd.ToggleIcon
-                    @icon='more-horizontal'
-                    @text={{t 'actions.overflow-options'}}
-                    @hasChevron={{false}}
-                    @size='small'
-                  />
-                  {{#if (can 'removeTags worker' @model)}}
-                    <dd.Interactive
-                      @text={{t 'resources.worker.tags.actions.remove'}}
-                      {{on 'click' (fn this.toggleRemoveModal B.data)}}
-                    />
+                {{#let
+                  (hash
+                    canEdit=(can 'setWorkerTags worker' @model)
+                    canRemove=(can 'removeTags worker' @model)
+                  )
+                  as |perms|
+                }}
+                  {{#if (or perms.canEdit perms.canRemove)}}
+                    <Hds::Dropdown as |dd|>
+                      <dd.ToggleIcon
+                        @icon='more-horizontal'
+                        @text={{t 'actions.overflow-options'}}
+                        @hasChevron={{false}}
+                        @size='small'
+                      />
+                      {{#if perms.canEdit}}
+                        <dd.Interactive
+                          @text={{t 'resources.worker.tags.edit.title'}}
+                          {{on 'click' (fn this.toggleEditModal B.data)}}
+                        />
+                      {{/if}}
+                      {{#if perms.canRemove}}
+                        <dd.Interactive
+                          @text={{t 'resources.worker.tags.actions.remove'}}
```

Comment:
design question- should we use color critical for this to stand out?

Reply:
oh yes, good catch